actk/steps/raw/raw.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from pathlib import Path
+from typing import Union
+
+import dask.dataframe as dd
+import pandas as pd
+from datastep import Step, log_run_params
+
+from ...constants import DatasetFields
+from ...utils import dataset_utils
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+# This is the merge of all other steps required fields.
+# Reasoning here is that the user will only want to upload the raw
+# if the user is doing a full pipeline run
+REQUIRED_DATASET_FIELDS = DatasetFields.AllExpectedInputs
+
+###############################################################################
+
+
+class Raw(Step):
+    def __init__(
+        self,
+        filepath_columns=[
+            DatasetFields.SourceReadPath,
+            DatasetFields.NucleusSegmentationReadPath,
+            DatasetFields.MembraneSegmentationReadPath,
+        ],
+        metadata_columns=[DatasetFields.FOVId],
+        **kwargs,
+    ):
+        super().__init__(
+            filepath_columns=filepath_columns,
+            metadata_columns=metadata_columns,
+            **kwargs,
+        )
+
+    @log_run_params
+    def run(self, dataset: Union[str, Path, pd.DataFrame, dd.DataFrame], **kwargs):
+        """
+        Simple passthrough to store the dataset in local_staging/raw.
+        This does not copy any the image files to local_staging/raw, only the manifest.
+        This is an optional step that will only run if you want to upload the raw data.
```

Comment:
have you tried actually pushing the upload to see if the raw data gets into quilt ok?

Reply:
Check the files pushed. They arrive just fine :)